spec/cypress/support/step_definitions/common-interaction.js

```diff
@@ -10,6 +10,12 @@ const elements = {
   "notification banner": "[data-test=notification-banner]",
   "edit admin link": "[data-test=edit-admin-link]",
   "delete button": "[data-test=delete-button]",
+  "create delivery partner button":
+    '.govuk-button:contains("new delivery partner")',
+  "delivery partner name": 'input[type="text"]',
+  "change name link": 'a:contains("Change name")',
+  "name input": '[name*="name"]',
+  "back button": '.govuk-button:contains("Back")',
```

Comment:
Are you not concerned this list is going to get unmanageable?

Reply:
i am concerned yes, but I also can't think of anything better at the moment

Reply:
We could split this object into a few more specific ones - input fields, buttons, text display elements. 

That would also help enforce naming conventions. At the moment we have a `name` which is an input for `full_name` and a `name_input` which is an input for `name`...

Reply:
That's a good shout - but I'd be a bit worried about separating things that are on the same page from each other

It's definitely better than what we have now though!

Reply:
I've made the names a bit more consistent in this PR and will go for some bigger improvements in another PR (I just can't think of what they should be at the moment)